src/__tests__/to-be-empty-dom-element.js

```diff
@@ -5,17 +5,33 @@ test('.toBeEmptyDOMElement', () => {
     <span data-testid="not-empty">
         <span data-testid="empty"></span>
         <svg data-testid="svg-empty"></svg>
-    </span>`)
+    </span>
+    <span data-testid="with-comment"><!-- This Comment --></span>
+    <span data-testid="with-multiple-comments"><!-- Comment1 --><!-- Comment2 --></span>
+    <span data-testid="with-element"><span></span></span>
+    <span data-testid="with-element-and-comment"><!--Comment--><span></span></span>
+    <span data-testid="with-whitespace"> </span>
+    <span data-testid="with-whitespace">Text</span>`)
```

Comment:
I guess the second one should not have the same test id, right? Also, this second element is probably not being used by the test at all.